packages/plugin-ext/src/common/uri-components.ts

```diff
@@ -18,6 +18,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { UNTITLED_SCHEME } from '@theia/core/lib/common';
```

Comment:
Yes, I think so. I has reverted these codes about `UNTITLED_SCHEME`. And try to run CI in my own theia repo,  it can run successful 

- [action1](https://github.com/yiliang114/theia/actions/runs/2575979658)
- [action2](https://github.com/yiliang114/theia/actions/runs/2575939827)